reference/content-types/link.rst

```diff
@@ -0,0 +1,68 @@
+Link
+======
+
+Description
+-----------
+
+The link content type is a universal link provider. It allows to either create urls directly from selected
+entities or to insert directly an external url. The configuration allows to enable ``anchor`` and ``target`` fields as
+well as the available entity types.
+
+The types can also be extended by adding new types to the ``linkTypeRegistry``.
```

Comment:
would link to the cookbook here 🙂 
https://docs.sulu.io/en/2.3/cookbook/link-provider.html